common/include/common/KVMFR.h

```diff
@@ -139,6 +139,7 @@ typedef struct KVMFRFrame
   uint32_t        damageRectsCount;   // the number of damage rectangles (zero for full-frame damage)
   FrameDamageRect damageRects[KVMFR_MAX_DAMAGE_RECTS];
   bool            blockScreensaver;   // whether the guest has requested to block screensavers
+  bool            requestActivation;  // whether the guest has requested activation since the last frame
```

Comment:
Struct changed, bump KVMFR version

Reply:
Oops, fixed.

Reply:
Note: In future we should replace these bools with a bit field so that future enhancements like this do not break protocol compatibility.